src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,36 @@
+import {ClothingStore, LocalBusiness, Organization, Thing, WithContext, PostalAddress} from 'schema-dts';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
+${JSON.stringify(json)}
+</script>`;
+}
+
+export const SchemaWrapper = (data: any) => {
+  return JsonLd<ClothingStore>({
+    '@context': 'https://schema.org',
+    '@type': 'ClothingStore',
+    "@id": data.document.streamOutput._site.uid,
+    name: data.document.streamOutput.name,
+    address: parseAddress(data),
+    description: data.document.streamOutput.description,
+    telephone: data.document.streamOutput.mainPhone,
+    image: data.document.streamOutput.photoGallery[0].image.url,
+    openingHoursSpecification: data.document.streamOutput.hours, // TODO: parse hours
+    paymentAccepted: data.document.streamOutput.paymentOptions,
+    logo: data.document.streamOutput.logo, // this entry will not show up if logo is nil 
+  });  
+}
+
+export const parseAddress = (data: any) => {
```

Comment:
Nice, I think a next step for us is figuring out what helpers like this we want to write. Address, Hours seem the big ones. But maybe we want something that handles core information (name, phone, etc.) that comes from profile fields. 

And potentially helpers for non-location standard entity types like person or product.